test/test_rohmu_dates.py

```diff
@@ -1,34 +0,0 @@
-"""
-rohmu test case
-
-Copyright (c) 2017 Ohmu Ltd
-See LICENSE for details
-"""
-from rohmu.dates import parse_timestamp
-
-import datetime
-import dateutil.tz
-import re
-
-
-def test_parse_timestamp() -> None:
```

Comment:
Is this test now gone completely?

Reply:
```
$ git switch main
Already on 'main'
Your branch is up to date with 'origin/main'.
$ diff test/test_rohmu_dates.py test/test_dates.py && echo equal
equal
```

Probably I should have been written this explicitly in the commit message.